etna/transforms/timestamp/holiday.py

```diff
@@ -60,19 +93,42 @@ def _get_column_name(self) -> str:
             return self.__repr__()
 
     def _fit(self, df: pd.DataFrame) -> "HolidayTransform":
```

Comment:
Delete this method.

Reply:
TypeError: Can't instantiate abstract class HolidayTransform with abstract method _fit. This happens if we delete this method

Reply:
My bad, I don't  see that it is abstract method. Okey, don't delete it)